examples/ping.nim

```diff
@@ -0,0 +1,21 @@
+import chronos, stew/byteutils
+import ../webrtc/udp_transport
+import ../webrtc/stun/stun_transport
+import ../webrtc/dtls/dtls_transport
+import ../webrtc/sctp/[sctp_transport, sctp_connection]
+
+proc main() {.async.} =
+  let laddr = initTAddress("127.0.0.1:4244")
+  let udp = UdpTransport.new(laddr)
+  let stun = Stun.new(udp)
+  let dtls = Dtls.new(stun)
+  let sctp = Sctp.new(dtls)
```

Comment:
If the final instance that matter is `Sctp`, maybe we could just have something like `Sctp.new(laddr)`?

Reply:
There will be something similar in the `datachannel` PR: `Webrtc.new(...)` will be used.
If we use something like `Sctp.new(laddr)`, it means we have to stop all transports if we choose to stop `Sctp`, which I don't like. I'd rather have a way to manage the whole stack by itself (with `Webrtc`) or to manage each part of the stack (like this example).